Prose/ProseLib/Sources/ProseCoreTCA/Types/Message.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+@_implementationOnly import ProseCoreClientFFI
+import SharedModels
+
+public enum MessageKind: Equatable {
+    case chat
+    case error
+    case groupChat
+    case headline
+    case normal
+}
```

Comment:
I don't know what this is, maybe add a short doc comment on each case?

Reply:
Or you can just say "Mirroring `ProseCoreClientFFI.MessageKind`.", and the doc lives there, it would be fine.

Reply:
See my comment about staying nimble.

If I may refer you to the Rust implementation for now: https://github.com/prose-im/prose-core-client/blob/master/prose_core_client_ffi/src/types/message.rs

I should definitely check why my documentation isn't present in the generated Swift code (in `_ProseCoreClientFFI`). As you can see it works for `BareJid`. 

It would also be cool if that worked: https://github.com/apple/swift-docc/issues/208

Reply:
Thank you're for the Rust link. I don't understand "Normal", but we'll see later 😉